metadata-ingestion/src/datahub/ingestion/api/auto_work_units/auto_ensure_aspect_size.py

```diff
@@ -90,7 +90,7 @@ def ensure_aspect_size(
         on GMS side and failure of the entire ingestion. This processor will attempt to trim suspected aspects.
         """
         for wu in stream:
-            logger.debug(f"Ensuring size of workunit: {wu.id}")
+            # logger.debug(f"Ensuring size of workunit: {wu.id}")
```

Comment:
Can you remove this if not needed?

Reply:
I wanted to keep it in so that it's easy to enable it for debugging. It also effectively serves as a comment explaining what's going on in the code, even when disabled